RecipeApp.Core/RecipeConverter.cs

```diff
@@ -0,0 +1,270 @@
+﻿using RecipeApp.Core.Exceptions;
+using RecipeApp.Core.Units;
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+
+namespace RecipeApp
+{
+    public class RecipeConverter
+    {
+        public static string ConvertRecipe(string recipe, bool isSiUnit)
+        {
+            if (isSiUnit)
+            {
+                return ConvertToSiUnits(recipe);
+            }
+            else
+            {
+                return ConvertToCookingUnits(recipe);
+            }
+        }
+
+        public static string ConvertToSiUnits(string recipe)
+        {
+            string[] words = recipe.Split(' ', '\n');
+
+            for (int index = 0; index < words.Length; index++)
+            {
+                try
+                {
+                    StandardiseCookingUnit(index, words);
+                }
+                catch (InvalidRecipeException ex)
+                {
+                    Console.WriteLine($"Skipping word, because: {ex.Message}");
+                }
+            }
+
+            return string.Join(" ", words);
+        }
+
+        //public static string MakeUnitsPretty(string recipe)
+        //{
+        //    string[] words = recipe.Split(' ', '\n');
+
+        //    for (int index = 0; index < words.Length; index++)
+        //    {
+        //        if (words[index] == "ml" && ParseDouble(words[index - 1], out double amount))
+        //        {
+        //            if (amount > 100)
+        //            {
+        //                words[index] = "l";
+        //                words[index - 1] = (amount / 100).ToString("F2");
+        //            }
+        //        }
+
+        //        var cookingUnit = words[index];
+        //        var multiplier = FindSiMultiplier(cookingUnit);
+        //        if (multiplier == -1) continue;
+
+        //        ParseDouble(words[index - 1], out amount);
+
+        //        words[index] = GetClosestCookingUnit(amount);
+        //        words[index - 1] = (amount * multiplier / Conversion.CookingUnits[words[index]]).ToString("F2");
+        //    }
+
+        //    return string.Join(" ", words);
+        //}
```

Comment:
Don't leave commented out code.

Reply:
Would you say only leave main / master branch clean of comments (and PRs actually) or would you also suggest removing those lines in develop / wip branches where you try different method implementations?

Reply:
I would say upon pushing- they should be clean. Sorry for a late response.

Reply:
Alright ty!